src/plugins/kibana_usage_collection/server/collectors/application_usage/rollups.test.ts

```diff
@@ -112,23 +132,42 @@ describe('rollDailyData', () => {
           id: 'appId:2020-01-01',
           attributes: {
             appId: 'appId',
+            viewId: undefined,
```

Comment:
can we test when this field does not exist at all?

Reply:
We do have some tests with `viewId` not defined at all. although `viewId: undefined` is identical.

Reply:
Perfect! Thanks!